tests/settings.py

```diff
@@ -0,0 +1,23 @@
+SECRET_KEY = 'not-for-production'
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+
+    'logger',
+)
+
+MIDDLEWARE_CLASSES = ()
+
+LANGUAGE_CODE = 'en-us'
```

Comment:
Given this is just to allow us to create migrations using `manage.py`, I don't think it matters at all.